components/Home/BuOlas/contractUtils.jsx

```diff
@@ -1,57 +1,8 @@
-import {
-  getContractAddress,
-  getBuolasContract,
-  getOlasContract,
-} from 'common-util/Contracts';
-import { MAX_AMOUNT, parseEther, sendTransaction } from 'common-util/functions';
+import { getBuolasContract } from 'common-util/Contracts';
+import { sendTransaction } from 'common-util/functions';
 
 const SIGNER_ADDRESS = '0x70997970C51812dc3A010C7d01b50e0d17dc79C8';
```

Comment:
What is this address used for? I know it's not a part of your changes, but seems strange to run revoke on some hardcoded EOA 🤔